package.json

```diff
@@ -11,50 +11,50 @@
     "vanilla/*"
   ],
   "scripts": {
-    "bootstrap": "lerna bootstrap",
-    "lerna": "lerna",
-    "clean": "lerna run --parallel clean",
-    "build": "lerna run --parallel build",
-    "lint": "lerna run --parallel lint:es",
-    "lint:fix": "lerna run --parallel lint:es:fix",
-    "tsc": "lerna run --parallel tsc",
-    "test": "lerna run --parallel test",
+    "clean": "npm run clean --workspaces --if-present",
+    "clean:all": "rm -rf node_modules && npm run clean:all --workspaces --if-present",
+    "build": "npm run build --workspaces --if-present",
+    "lint": "npm run lint --workspaces --if-present",
+    "lint:fix": "npm run lint:fix --workspaces --if-present",
+    "tsc": "npm run tsc --workspaces --if-present",
+    "test": "npm run test --workspaces --if-present",
     "uberext": "ts-node ./scripts/create_uber_extension_project.ts",
     "update-packages": "ts-node ./scripts/update_packages.ts",
-    "dev:demo-core-sdk": "yarn workspace demo-core-sdk develop",
```

Comment:
Honestly, we should probably just delete these. I think it's an anti-pattern to be having your root package.json manage scripts for workspaces.